packages/routes-gen/src/route.test.ts

```diff
@@ -23,3 +31,8 @@ it("only replaces the first occurrence when param names share the same prefix",
     route("/podcasts/:topicId/:topic", { topic: "sports", topicId: "123" })
   ).toEqual("/podcasts/123/sports");
 });
+
+it('escapes the path segments', () => {
+  expect(route('/contacts/:contact_id.vcard', { contact_id: '123' })).toEqual('/contacts/123.vcard');
```

Comment:
@sandulat actually, we probably shouldn't support this yet as the exports should be updated in remix for escapes there's currently no way to tell what we're escaping because the `[]` are removed from routes-gen routes.d.ts